src/murmur/CMakeLists.txt

```diff
@@ -240,11 +240,8 @@ if(grpc)
 
 	target_compile_definitions(murmur PRIVATE "USE_GRPC")
 	target_include_directories(murmur PRIVATE ${CMAKE_CURRENT_BINARY_DIR})
-	target_link_libraries(murmur PRIVATE gRPC::grpc++_reflection)
```

Comment:
@davidebeatrici is there a specific reason we were linking against `grpc++_reflection` only instead of against `grpc++`?

Reply:
If I recall correctly it exports some symbols we need.